MCprep_addon/spawner/spawn_util.py

```diff
@@ -125,6 +137,29 @@ def draw_item(self, context, layout, data, set, icon, active_data, active_propna
 				layout.label(text="", icon='QUESTION')
 
 
+#class MCPREP_UL_models(bpy.types.UIList):
```

Comment:
Does this section need to be deleted, if just commented out? Or was this resolved in a more recent (not yet pushed?) update?

Reply:
While I was fixing bugs, I commented out some stuff, I still need to clean up. Thanks for reminding me